.percy.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+snapshot:
+  widths:
+    - 375
+    - 768
+    - 1280
+  minHeight: 1024
+discovery:
+  allowedHostnames: []
+  networkIdleTimeout: 100
```

Comment:
Is this configured according to our breakpoints? (I'm wondering how much it will need to change for other sites)

Reply:
yeah, it uses the values from cdhweb sass sources. fwiw i think these breakpoints are different from most other projects, but the others are all the same — at least ppa and s&co definitely are; not sure about derrida